P5/Source/Guidelines/en/SA-LinkingSegmentationAlignment.xml

```diff
@@ -3242,41 +3199,130 @@ will only select the text of the poem, with no markup inside.</p>
+      encode the same information, they do not attempt to reproduce it comprehensively. TEI
+      annotations should be able to be exported to WADM's JSON-LD format, and WADM to be imported
+      into TEI, but in the latter case, there may be some information loss.</p>
+    <p>Annotations have a target, an optional body, and may contain metadata specifying the person
+      or agent responsible for lifecycle events, the nature of those events, and the license of the
+      annotation. Targets are expressed using the <att>target</att> attribute on the
+        <gi>annotation</gi>, and at least one target is required. Bodies may be expressed using
+        <gi>ptr</gi> or <gi>ref</gi> with <att>target</att> if they are URIs, or using <gi>note</gi>
+      for a textual body. Lifecycle metadata may be encoded using <gi>respStmt</gi> and
+        <gi>revisionDesc</gi>, while licenses use <gi>license</gi>.</p>
+    <p>The example below illustrates how standoff annotations can be used to connect words in a text
+      with <gi>place</gi> elements in a list. The words <q>Gallia</q> and <q>Galliae</q> in the
+      edition are connected by an annotation in the standOff section which points to them using
+        <code>string-range()</code> pointers and references the defintion of the place (also in the
+      standOff section). <egXML xmlns="http://www.tei-c.org/ns/Examples">
+        <TEI>
+          <teiHeader>
+            <fileDesc>
+              <titleStmt>
+                <title>Title</title>
+              </titleStmt>
+              <publicationStmt>
+                <p>Publication Information</p>
+              </publicationStmt>
+              <sourceDesc>
+                <p>Information about the source</p>
+              </sourceDesc>
+            </fileDesc>
+          </teiHeader>
+          <text>
+            <body>
+              <div type="edition">
+                <div type="textpart" subtype="chapter" n="1" xml:id="ch1">
+                  <p n="1" xml:id="c1p1">
+                    <seg n="1" xml:id="c1p1s1">Gallia est omnis divisa in partes tres, quarum unam
+                      incolunt Belgae, aliam Aquitani, tertiam qui ipsorum lingua Celtae, nostra
+                      Galli appellantur.</seg>
+                    <!-- ... -->
+                    <seg n="6" xml:id="c1p1s6">Belgae ab extremis Galliae finibus oriuntur,
+                      pertinent ad inferiorem partem fluminis Rheni, spectant in septentrionem et
+                      orientem solem.</seg>
+                    <seg n="7" xml:id="c1p1s7">Aquitania a Garumna flumine ad Pyrenaeos montes et
+                      eam partem Oceani quae est ad Hispaniam pertinet; spectat inter occasum solis
+                      et septentriones.]</seg>
+                  </p>
+                </div>
+              </div>
+            </body>
+          </text>
+          <TEI type="annotations">
```

Comment:
I wonder if this first example should show `<standoff>` as a sibling of `<text>`? I think it's a gentler introduction.